docs/docs/playground/basics/index.tsx

```diff
@@ -0,0 +1,18 @@
+import styled, { css } from "styled-components";
```

Comment:
IMO we shouldn't default to `index.ts[x]` for first files, I personally find it harder to navigate a codebase with a large number of `index` filenames